src/panels/lovelace/cards/hui-markdown-card.ts

```diff
@@ -0,0 +1,90 @@
+import { html, LitElement } from "@polymer/lit-element";
+import { classMap } from "lit-html/directives/classMap.js";
+
+import "../../../components/ha-card.js";
+import "../../../components/ha-markdown.js";
+
+import { LovelaceCard, LovelaceConfig } from "../types.js";
+
+interface Config extends LovelaceConfig {
+  content: string;
+  title?: string;
+}
+
+export class HuiMarkdownCard extends LitElement
+  implements LovelaceCard {
+  protected config?: Config;
+
+  static get properties() {
+    return {
+      config: {},
+    };
+  }
+
+  public getCardSize() {
+    return this.config!.content.split("\n").length;
+  }
+
+  public setConfig(config: Config) {
+    if (!config.content) {
+      throw new Error("Invalid Configuration: Content Required");
+    }
+
+    this.config = config;
+  }
+
+  protected render() {
+    if (!this.config) {
+      return html``;
+    }
+
+    return html`
+      ${this.renderStyle()}
+      <ha-card .header="${this.config.title}">
+        <ha-markdown class="markdown ${classMap({
```

Comment:
Lets put each attribute/property on its own line [example](https://github.com/home-assistant/home-assistant-polymer/blob/master/src/panels/lovelace/cards/hui-entity-button-card.ts#L74)

Reply:
This is only if there is multiple. So `ha-card` with `.header` only is okay